ddl/ddl.go

```diff
@@ -725,10 +730,31 @@ func (d *ddl) CreateTable(ctx context.Context, ident ast.Ident, colDefs []*ast.C
 	}
 
 	err = d.startDDLJob(ctx, job)
+	if err == nil {
+		err = d.handleTableOptions(options, tbInfo, schema.ID)
```

Comment:
This may not work on distributed KV engine.
At this point, the table may not have been created.